opencensus/trace/file_exporter.py

```diff
@@ -37,13 +37,13 @@ class FileExporter(base_exporter.Exporter):
 
     :type file_mode: str
     :param file_mode: The file mode to open the output file with.
-                      Defaults to w+
+                      Defaults to a.
 
     """
 
     def __init__(self, file_name=DEFAULT_FILENAME,
                  transport=sync.SyncTransport,
-                 file_mode='w+'):
+                 file_mode='a'):
```

Comment:
Nice, this actually aligns better with log handler `class logging.FileHandler(filename, mode='a', encoding=None, delay=False)`.